bpfman/src/types.rs

```diff
@@ -653,6 +836,16 @@ impl ProgramData {
      * Methods for setting and getting kernel information.
      */
 
+    /// Retrieves the name of the kernel.
```

Comment:
```suggestion
    /// Retrieves the name of the program.
```

Sorry this is a bit confusing but please see the API documentation here https://docs.aya-rs.dev/aya/programs/struct.programinfo for all these fields

All APIs with the "kernel" substring in the name are just pulling from the aya ProgramInfo structure